src/components/navigation/navigation-list/NavigationList.tsx

```diff
@@ -95,6 +95,7 @@ export class NavigationList extends Component<Props> {
               onNext={() => this._handleDownKeyPressed(index)}
               onPrev={() => this._handleUpKeyPressed(index)}
               dispatcher={this.props.dispatcher}
+              slideNumber={index}
```

Comment:
could you remove in the PR since `this._itemsRefMap` not uses anymore
```
              ref={node => {
                this._setNavigationItemRef(index, node);
              }}
```
```
  private _setNavigationItemRef = (index: number, ref: NavigationItem | null) => {
    return this._itemsRefMap.set(index, ref);
  };
```
```
  componentWillUnmount() {
    this._itemsRefMap = new Map();
  }
```
```
private _itemsRefMap: Map<number, NavigationItem | null> = new Map();
```

Reply:
also, be aware, since `withText` wrapper wraps component, the `ref` of the wrapped component can't be reached anymore.